lib/controllers/game_controller.cpp

```diff
@@ -269,12 +271,26 @@ void GameController::setupInput() {
     inputMapping[4][InputKeyCode::EController_x] = PlayerInput::SHOOT;
     inputMapping[4][InputKeyCode::EController_b] = PlayerInput::GRAB;
 
+    // Cheats
+    for (int i = 0; i < 4; i++) {
+        inputMapping[i][InputKeyCode::EKey_f1] = PlayerInput::SKIP_LEVEL;
+        inputMapping[i][InputKeyCode::EKey_f2] = PlayerInput::KILL_EVERYONE_EXCEPT_YOURSELF;
+        inputMapping[i][InputKeyCode::EKey_f3] = PlayerInput::INFINITE_HEALTH;
+        inputMapping[i][InputKeyCode::EKey_f4] = PlayerInput::RANDOM_WEAPON;
+        inputMapping[i][InputKeyCode::EKey_f5] = PlayerInput::LASER_WEAPON;
+    }
```

Comment:
It could cause problems. What if for instance player 1 does not exists?  Then you would not be able to apply the "Kill all other" or a weapon drop cheat. So In the system I am retrieving the lowest player number.